test/saml20.tests.js

```diff
@@ -138,13 +139,75 @@ describe('saml 2.0 assertion', function () {
 
   });
 
+
+
+  it('should should allow expired cert if option not passed', function (done) {
+
+    options.lifetimeInSeconds = 10000;
+
+    var signedAssertion = saml20.create(options);
+    var publicKey = fs.readFileSync(__dirname + '/test-auth0.cer').toString();
+    // The embedded cert is expired, so we can use this as is.
+    const samlPassport = new SamlPassport({cert: publicKey, realm: 'urn:myapp', checkRecipient: false});
+    samlPassport.validateSamlAssertion(signedAssertion, function(err, profile) {
+      should.not.exists(err);
+      // err.message.should.equal('The signing certificate is not currently valid.');
```

Comment:
Any reason for this to hang out?

Reply:
No, just some copy-pasted cruft. Removing now. Thanks for catching that.

Reply:
Removed (and resolving, if that's not presumptuous on my part)